test/integration/generated/datafactory/src/dataFactoryClient.ts

```diff
@@ -52,13 +53,18 @@ export class DataFactoryClient extends coreClient.ServiceClient {
 
   /**
    * Initializes a new instance of the DataFactoryClient class.
+   * @param credentials Subscription credentials which uniquely identify client subscription.
    * @param subscriptionId The subscription identifier.
    * @param options The parameter options
    */
   constructor(
+    credentials: coreAuth.TokenCredential,
```

Comment:
Looks like some default setting changed and now everything is getting a credential?